### ISSUE TITLE
Jupyter config error

### ISSUE THREAD

**Issue by KumoLiu** [Project-MONAI/MONAI#5777]
**Describe the bug**
When running jupyter notebook in the 1.1.0 container, report error below:
```
[C 08:47:06.307 NotebookApp] Bad config encountered during initialization: The 'contents_manager_class' trait of a NotebookApp instance expected a subclass of notebook.services.contents.manager.ContentsManager or jupyter_server.contents.services.managers.ContentsManage, not the JupytextContentsManager JupytextContentsManager.
```

**To Reproduce**
1. docker pull projectmonai/monai:1.1.0
2. create a new container 
3. run into container
4.  run `jupyter notebook`

**Environment**
```
MONAI version: 1.1.0
Numpy version: 1.22.2
Pytorch version: 1.13.0a0+d0d6b1f
MONAI flags: HAS_EXT = True, USE_COMPILED = False, USE_META_DICT = False
MONAI rev id: a2ec3752f54bfc3b40e7952234fbeb5452ed63e3
MONAI __file__: /opt/monai/monai/__init__.py

Optional dependencies:
Pytorch Ignite version: 0.4.10
Nibabel version: 4.0.2
scikit-image version: 0.19.3
Pillow version: 9.0.1
Tensorboard version: 2.10.1
gdown version: 4.6.0
TorchVision version: 0.14.0a0
tqdm version: 4.64.1
lmdb version: 1.3.0
psutil version: 5.9.2
pandas version: 1.4.4
einops version: 0.6.0
transformers version: 4.21.3
mlflow version: 2.0.1
pynrrd version: 1.0.0

For details about installing the optional dependencies, please visit:
    https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies


================================
Printing system config...
================================
System: Linux
Linux version: Ubuntu 20.04.5 LTS
Platform: Linux-5.15.0-56-generic-x86_64-with-glibc2.10
Processor: x86_64
Machine: x86_64
Python version: 3.8.13
Process name: python
Command: ['python']
Open files: [popenfile(path='/root/.vscode-server/data/logs/20221220T084448/ptyhost.log', fd=19, position=0, mode='a', flags=33793)]
Num physical CPUs: 12
Num logical CPUs: 20
Num usable CPUs: 20
CPU usage (%): [3.7, 1.5, 4.1, 0.9, 3.8, 1.0, 4.0, 0.7, 10.7, 0.9, 6.4, 0.3, 5.6, 0.3, 5.4, 0.4, 1.9, 1.3, 0.7, 0.7]
CPU freq. (MHz): 3505
Load avg. in last 1, 5, 15 mins (%): [2.4, 1.9, 1.0]
Disk usage (%): 78.9
Avg. sensor temp. (Celsius): UNKNOWN for given OS
Total physical memory (GB): 62.6
Available memory (GB): 58.4
Used memory (GB): 3.5

================================
Printing GPU config...
================================
Num GPUs: 2
Has CUDA: True
CUDA version: 11.8
cuDNN enabled: True
cuDNN version: 8600
Current device: 0
Library compiled for CUDA architectures: ['sm_52', 'sm_60', 'sm_61', 'sm_70', 'sm_75', 'sm_80', 'sm_86', 'sm_90', 'compute_90']
GPU 0 Name: NVIDIA GeForce RTX 3090 Ti
GPU 0 Is integrated: False
GPU 0 Is multi GPU board: False
GPU 0 Multi processor count: 84
GPU 0 Total memory (GB): 23.7
GPU 0 CUDA capability (maj.min): 8.6
GPU 1 Name: NVIDIA GeForce RTX 3090 Ti
GPU 1 Is integrated: False
GPU 1 Is multi GPU board: False
GPU 1 Multi processor count: 84
GPU 1 Total memory (GB): 23.7
GPU 1 CUDA capability (maj.min): 8.6
```

**Additional context**
MONAI docker 1.0.1 works well.

**Comment by wyli** [comment]
update

the current workaround is to append 
```bash
--NotebookApp.contents_manager_class='notebook.services.contents.largefilemanager.LargeFileManager'`
```
when starting `jupyter notebook` or `jupyter lab`

**Comment by Nic-Ma** [comment]
Hi @wyli @KumoLiu ,

As the latest 22.11 docker is already available, is this issue fixed in it?
nvcr.io/nvidia/pytorch:22.11-py3
And BTW, we may need to release a minor version later to upgrade the numpy dependency and base docker?

Thanks.

**Comment by KumoLiu** [comment]
> Hi @wyli @KumoLiu ,
> 
> As the latest 22.11 docker is already available, is this issue fixed in it? nvcr.io/nvidia/pytorch:22.11-py3 And BTW, we may need to release a minor version later to upgrade the numpy dependency and base docker?
> 
> Thanks.

Hi @Nic-Ma,

Yes, have tested it in 22.11, and it works well.

Thanks!

**Comment by wyli** [comment]
I've rebuilt image `projectmonai/monai:1.1.0` to include a config workaround:

```bash
/opt/monai# cat /root/.jupyter/jupyter_notebook_config.py 
c.NotebookApp.contents_manager_class = 'notebook.services.contents.largefilemanager.LargeFileManager'
```

now the 1.1.0 image doesn't have this issue. dev branch should be fixed by upgrading to at least pytorch 22.11...
